File: BOJ/반복문/10818.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n=sc.nextInt();

        int max, min;
        max = sc.nextInt();
        min = max; // max와 min을 첫 번째 숫자 값으로 초기화
        n--;

        while(n>0){ // n개의 숫자 입력 받으면서 max와 min을 갱신
            int num = sc.nextInt();
            if(num>=max) max=num;
            else if(num<=min) min=num;
            n--;
        }

        System.out.println(min + " " + max);
    }
}

File: BOJ/반복문/10818.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
	ios::sync_with_stdio(0);
	cin.tie(0);

	int n, max, min;
	cin >> n;
	cin >> max;
	min = max;

	while (--n) { // 위에서 첫 번째 수를 미리 읽었기 때문에 --n
		int num;
		cin >> num;
		if (num >= max) max = num;
		else if (num <= min) min = num;
	}

	cout << min << " " << max;
}

// 1. max, min을 첫번째 수로 초기화한다.
// 2. n개의 수를 읽어 max와 min을 갱신한다.
